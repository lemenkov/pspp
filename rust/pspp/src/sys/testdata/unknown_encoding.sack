# File header.
"$FL2"; s60 "$(#) SPSS DATA FILE PSPP synthetic test file";
2; # Layout code
4; # Nominal case size
0; # Not compressed
0; # Not weighted
0; # No cases.
100.0; # Bias.
"01 Jan 11"; "20:53:52";
"PSPP synthetic test file: "; i8 244; i8 245; i8 246; i8 248; s34 "";
i8 0 *3;

# Numeric variables.
2; 0; 0; 0; 0x050800 *2; s8 "A";
2; 0; 0; 0; 0x050800 *2; s8 "B";
2; 0; 0; 0; 0x050800 *2; s8 "C";
2; 0; 0; 0; 0x050800 *2; s8 "D";

# Dictionary termination record.
999; 0;
