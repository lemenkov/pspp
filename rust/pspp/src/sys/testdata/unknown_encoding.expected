This system file does not indicate its own character encoding.  For best results, specify an encoding explicitly.  Use SYSFILE INFO with ENCODING="DETECT" to analyze the possible encodings.

Text string contains invalid bytes for UTF-8 encoding: "PSPP synthetic test file: ����                                  "

╭──────────────────────┬────────────────────────╮
│       Created        │    01-JAN-2011 20:53:52│
├──────────────────────┼────────────────────────┤
│Writer Product        │PSPP synthetic test file│
├──────────────────────┼────────────────────────┤
│       Compression    │None                    │
│       Number of Cases│                       0│
╰──────────────────────┴────────────────────────╯

╭─────────┬──────────────────────────────╮
│Label    │PSPP synthetic test file: ����│
│Variables│                             4│
╰─────────┴──────────────────────────────╯

╭─┬────────┬─────┬─────────────────┬─────┬─────┬─────────┬────────────┬────────────┬──────────────╮
│ │Position│Label│Measurement Level│ Role│Width│Alignment│Print Format│Write Format│Missing Values│
├─┼────────┼─────┼─────────────────┼─────┼─────┼─────────┼────────────┼────────────┼──────────────┤
│a│       1│     │                 │Input│    8│Right    │F8.0        │F8.0        │              │
│b│       2│     │                 │Input│    8│Right    │F8.0        │F8.0        │              │
│c│       3│     │                 │Input│    8│Right    │F8.0        │F8.0        │              │
│d│       4│     │                 │Input│    8│Right    │F8.0        │F8.0        │              │
╰─┴────────┴─────┴─────────────────┴─────┴─────┴─────────┴────────────┴────────────┴──────────────╯
